addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/VoiceRSSTTSService.java

```diff
@@ -164,22 +164,37 @@ public AudioStream synthesize(String text, Voice voice, AudioFormat requestedFor
         return audioFormats;
     }
 
-    /**
-     * Up to now only MP3 supported.
-     */
-    private final AudioFormat getAudioFormat(String format) {
-        // MP3 format
-        if (AudioFormat.CODEC_MP3.equals(format)) {
-            // we use by default: MP3, 44khz_16bit_mono
+    private final AudioFormat getAudioFormat(String apiFormat) {
+        if (apiFormat.equals("MP3")) {
+            // we use by default: MP3, 44khz_16bit_mono with bitrate 64 kbps
             Boolean bigEndian = null; // not used here
             Integer bitDepth = 16;
-            Integer bitRate = null;
+            Integer bitRate = 64000;
```

Comment:
Yes, that's true, that is the values we decided to use for the API.
I suppose that you expect constants ?